src/ldp.rs

```diff
@@ -493,6 +494,34 @@ async fn sign_proof(
     Ok(proof)
 }
 
+async fn sign_nojws(
+    document: &(dyn LinkedDataDocument + Sync),
+    options: &LinkedDataProofOptions,
+    _resolver: &dyn DIDResolver,
```

Comment:
Is it just to have consistent interfaces that you keep `resolver`?

Reply:
Yes.
Theoretically `to_jws_payload` could be updated to take the `resolver` and use it to load remote JSON-LD contexts over DID URLs.